README.md

```diff
@@ -1,36 +1,23 @@
-# NATS Roadmap
+# General NATS inforamtion
 
-This is the [NATS.io](https://nats.io/) product roadmap for some of the key features our team is planning to deliver.  Only high level features and issues are listed; we’ll certainly continue to work on minor enhancements that are not listed here.
+This repository has general nats information and hosts various documents and
+reports.  Be sure to visit [NATS.io](https://nats.io) for the most up to date
+information.
 
-If you have any questions or comments on these items, please let us know via GitHub [here](https://github.com/nats-io/nats-general/issues/new).  You can also always communicate through Slack or Google Group with us on our [community page](https://www.nats.io/community) - we look forward to hearing from you!
-
-__2018 features and enhancements:__
-
-* NATS Server [Cluster Gateway](https://github.com/nats-io/gnatsd/issues/452)
-* [NKey](https://github.com/nats-io/nkeys) support across maintainer supported clients
-* [Drain mode](https://github.com/nats-io/go-nats/pull/378) support across maintainer supported clients
-* Support for multiple isolated accounts in the same server
-* Decentralized Authentication and Authorization using JWTs and NKeys
+## NATS Roadmap
 
-__2019 Future items to be planned:__
+Here is the NATS product roadmap with some of the key features we are planning to deliver.  Only high level features and issues are listed; we’ll certainly continue to work on minor enhancements that are not mentioned here.
```

Comment:
reword the first sentence?
The NATS Product Roadmap shows some of the key features being planned for future delivery.